### PR TITLE
*: Add ClusterProfileAWSArm64

### PR DESCRIPTION
**Pull request by LorbusChris**
/cc @jcpowermac

### CODE REVIEW

File: pkg/api/types.go [hunk 1]
```diff
@@ -1111,6 +1115,8 @@ func (p ClusterProfile) LeaseType() string {
 		ClusterProfileAWSCentos40,
 		ClusterProfileAWSGluster:
 		return "aws-quota-slice"
+	case ClusterProfileAWSArm64:
+		return "aws-arm64-quota-slice"
```
Comment (LorbusChris) [inline pkg/api/types.go:1119]: Boskos will have to learn about this

**Review by deepsm007 — APPROVED** [review]

**Comment by LorbusChris** [comment]
Not sure this needs anything else, lifting WIP for now.

File: cmd/repo-init/main.go [hunk 2]
```diff
@@ -636,6 +636,8 @@ func determineWorkflowFromClusterPorfile(clusterProfile api.ClusterProfile) *str
 	switch clusterProfile {
 	case api.ClusterProfileAWS:
 		ret = "ipi-aws"
+	case api.ClusterProfileAWSArm64:
+		ret = "ipi-aws"
```
Comment (LorbusChris) [inline cmd/repo-init/main.go:640]: note that I'm planning to adapt the `ipi-aws` workflow to work with arm64. see https://github.com/openshift/release/pull/19631

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**